Sources/MapboxMaps/Foundation/Camera/CameraAnimator.swift

```diff
@@ -40,27 +58,58 @@ public class CameraAnimator: NSObject {
     // MARK: Initializer
     internal init(delegate: CameraAnimatorDelegate,
                   propertyAnimator: UIViewPropertyAnimator,
```

Comment:
I'm curious why the property animator is injected

Reply:
I think this was done so as to avoid having all the different convenience initializers for `UIViewPropertyAnimator` as initializers of the `CameraAnimator`..